export_dagmc_cmd/README.md

```diff
@@ -0,0 +1,24 @@
+```
+mkdir bld
+cd bld
+cmake .. -DCMAKE_PREFIX_PATH=/path/to/Trelis-16.0/bin -DCMAKE_INSTALL_BINARY_DIR=bin -DMOAB_DIR=/path/to/MOAB/lib
```

Comment:
I found that the `-DCMAKE_INSTALL_BINARY_DIR` variable wasn't necessary, but I don't have proof that this is always true.